pkg/k8s/meta.go

```diff
@@ -41,6 +42,10 @@ func GetNamespacedName(obj metav1.Object) types.NamespacedName {
 	}
 }
 
+func IsOperatorCRD(gvk schema.GroupVersionKind) bool {
+	return strings.HasSuffix(gvk.Group, operatork8s.CoreCNRMGroup)
+}
+
 func IsManagedByKCC(gvk schema.GroupVersionKind) bool {
 	return strings.HasSuffix(gvk.Group, CNRMGroup)
 }
```

Comment:
Could we fold the function IsOperatorCRD into IsManagedByKCC? IOW, IsManagedByKCC should say "false" for the operator CRD, right? It would also be nice to get some cheap unit tests on IsManagedByKCC func

Reply:
Good idea! Done.